zebra/zebra_fpm_netlink.c

```diff
@@ -169,9 +169,12 @@ static int netlink_route_info_add_nh(struct netlink_route_info *ri,
 	struct interface *ifp = NULL, *link_if = NULL;
 	struct zebra_if *zif = NULL;
 	vni_t vni = 0;
+	extern struct in_addr ipv4_ll;
+	static union g_addr ipv4ll_gateway = {};
 
 	memset(&nhi, 0, sizeof(nhi));
 	src = NULL;
+	ipv4ll_gateway.ipv4 = ipv4_ll;
```

Comment:
again, this is sort of awkward - we don't need to be re-initializing this every time '...add_nh()' is called?